src/game.cpp

```diff
@@ -1700,7 +1700,7 @@ bool game::cancel_activity_or_ignore_query(const char *reason, ...)
     va_end(ap);
 
     bool force_uc = get_option<bool>( "FORCE_CAPITAL_YN" );
-    int ch = (int)' ';
+    int ch;
```

Comment:
Casting the space character is kind of wacky, but this shuld be initialized to something.

Reply:
Literally all the initialization did was assign the value and then immediately overwrite it courtesy of the do-while below it (which is why it got flagged; a "value is assigned but never used" weak warning). How about just moving it below the stop_message statement to make that connection a bit more apparent?

Reply:
Variables should be initialized by default, it's beter to have "useless" initializations than to risk missing any.